addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8026,7 +8026,11 @@ msgctxt "#15064"
 msgid "Apply changes directly"
 msgstr ""
 
-#empty string with id 15065
+#. Warning if no dsp modes are present
+#: xbmc/settings/dialogs/GUIDialogAudioDSPManager.cpp
+msgctxt "#15065"
+msgid "No audio DSP modes present!"
```

Comment:
It means "No audio DSP modes available", to inform user during open of manager dialog and close them directly after OK Dialog with this message.

Better change it to your given text?